### ISSUE TITLE
Non English content embedding web part?

### ISSUE THREAD

**Issue by iparlade** [pnp/custom-learning-office-365#500]
Hello, we deployed the solution to a Spanish tenant, taking care of all the multi-lingual issues.
So far so good, when browsing the solution the correct Spanish text and translations (where available) appear.
However, when we add the LP App to another site and use the web part to embed content inside a page, only the English content appears, no matter the language used in the browser.

Using the web part would be extremely useful for us, but we need it to show the Spanish language version.

Is this a well known issue? Does it have a workaround? I haven't been able to find anything about it, and the web part controls allow me to filter content but not change the language.

EDIT: Now it show the content in Spanish sometimes, upon page reload, but seems quite inconsistent.
Any idea?

Any help would be appreciated.

Thanks.

Regards,
Ignacio

**Comment by jnewgard-thrive** [comment]
Hi @iparlade 

Could you confirm what type of site the Learning Pathways Web Part was added to? (Communication, Team, Etc)

Additionally, do the users have their language set within their Office Profiles?

[Change Your Display Language](https://support.microsoft.com/en-us/topic/change-your-display-language-and-time-zone-in-microsoft-365-for-business-6f238bff-5252-441e-b32b-655d5d85d15b?ui=en-us&rs=en-us&ad=us)

**Comment by iparlade** [comment]
Hello,
I can confirm a Communication Site was created.
Regarding the language, my own user had none (as per the install guide in multilingual scenarios), but other users had Spanish and saw contents in English. Could it be that the web part defaulted to English when I embedded it, and other users browsing the site got stuck with English language even if they had Spanish because of that?

I have modified my profile, adding Spanish, and will do more testing with other users tomorrow to try and see if the language gets changed again.

Thank you for your help.

**Comment by iparlade** [comment]
Just checked, and all the users (including me) see the embedded web part in English.

I can confirm all my language settings are set to Spanish:
![image](https://user-images.githubusercontent.com/2000571/115835139-3fb51580-a416-11eb-8345-78ac5b13f550.png)

![image](https://user-images.githubusercontent.com/2000571/115835389-81de5700-a416-11eb-94c8-d038a6f73c20.png)

Any idea?

Thanks!

**Comment by juliemturner** [comment]
The web part isn't configurable for language it picks up the language setting for the user context from the page... that language setting is first based on your delve profile setting, second based on your browser setting, and third based on the language of the site collection. That said there are a lot of variables here and lots of those are related to how SharePoint multilingual works. 

Questions that come to mind:
- Is the site collection where you've put the web part configured for multilingual.
- Is the UX of the site all shown in Spanish and just the contents of the web part are in English?
- Can you confirm what the contents that are displayed are, I'm interested if you're at a menu or if you've filtered the web part to a particular playlist?

**Comment by iparlade** [comment]
Hello, after some investigation, I think I know what is happening, at least partially.
The communication site where the web part was used was created with the lookbook provisioning. It did set the default language to English, and did not enable multilingual.

After enabling multilingual, I noticed that it was not picking the language configured for my account (or others). The same was happening with the learning pathways site. Once I chose to view the page in Spanish selecting it in the language list of the site, all the places where the web app was being used appeared in Spanish. Thus, I don't know if the automatic language selection is working at all. The language is set to Spanish everywhere, and all the rest of the users have it as well.

As we are only interested in the Spanish version, I have set a workaround. I have configured the default site to be the es-es version, and then everything shows in Spanish (as it should).

These sites are only temporary, while the migration lasts, so I am not too bothered by the workaround.

Thanks again
